README.md

```diff
@@ -172,3 +182,21 @@ the API documentation in [toxcore/tox.h](toxcore/tox.h) for more information.
 
 - [Another echo bot](https://wiki.tox.chat/developers/client_examples/echo_bot)
 - [minitox](https://github.com/hqwrong/minitox) (A minimal tox client)
+
+## SAST Tools
+
+- [clang-tidy](https://clang.llvm.org/extra/clang-tidy/): A clang-based C++
+  "linter" tool.
+- [Coverity](https://scan.coverity.com/): Find and fix defects in your Java,
```

Comment:
The sentence tense/structure is not like the others. Every other tool is "A X for Y".

Perhaps "A cloud-based static analyzer service for Java, C/C++, C#, JavaScript, Ruby, or Python that is free for open source projects"?

Reply:
I copied it from their website.